imports/ui/components/Message.jsx

```diff
@@ -83,11 +83,34 @@ class Messages extends Component {
 			return (
 				<div className='conversation-container'>
 					<Conversation individualConversations={this.props.messages.activeThread.messagesWithContact} userId={this.props.currentUser._id}/>
-					<input type="text"/>
+					<input type="text" onKeyDown={this.inputKeyPress}/>
 				</div>
 			)
 		}
 	}
+
+	inputKeyPress(e){
+	   if (e.keyCode == 13){
+	   		const activeThread = this.props.messages.activeThread
```

Comment:
please add semi colon. I know it's not required in JS but I want to keep the code clean. Also make sure this is indented correctly